metacatalog/data/licenses.csv

```diff
@@ -1,4 +1,9 @@
+3,CC BY 4.0,Creative Commons Attribution 4.0 International,"You are free to: Share — copy and redistribute the material in any medium or format; Adapt — remix, transform, and build upon the material; for any purpose, even commercially. Under the following terms: Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.  ShareAlike — If you remix, transform, or build upon the material, you must distribute your contributions under the same license as the original.",,https://creativecommons.org/licenses/by/4.0/legalcode.txt,True,False,Ture
+4,CC BY-SA 4.0,Creative Commons Attribution-ShareAlike 4.0 International,"You are free to: Share — copy and redistribute the material in any medium or format; Adapt — remix, transform, and build upon the material; for any purpose, even commercially. Under the following terms: Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.  ShareAlike — If you remix, transform, or build upon the material, you must distribute your contributions under the same license as the original.",,https://creativecommons.org/licenses/by-sa/4.0/legalcode.txt,True,True,True
+5,CC BY-NC 4.0,Creative Commons Attribution-NonCommerical 4.0 International,"You are free to: Share — copy and redistribute the material in any medium or format Adapt — remix, transform, and build upon the material. Under the following terms: Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.  NonCommercial — You may not use the material for commercial purposes.",,https://creativecommons.org/licenses/by-nc/4.0/legalcode.txt ,True,False,False
+6,CC BY-ND 4.0,Creative Commons Attribution-NoDerivatives 4.0 International,"You are free to: Share — copy and redistribute the material in any medium or format for any purpose, even commercially. Under the following terms: Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.  NoDerivatives — If you remix, transform, or build upon the material, you may not distribute the modified material.",,https://creativecommons.org/licenses/by-nd/4.0/legalcode.txt ,True,False,True
```

Comment:
@sihassl , I think we should really drop this license, see the comment of the pull request.